data/sound-strings/zh-Hans.json/localize.json

```diff
@@ -30,6 +30,7 @@
 "in_2_5_kilometers":"前方两点五千米",
 "in_3_kilometers":"前方三千米",
 "then":"然后",
+"dist_direction_onto_street":"%1$s %2$s 上 %3$s",
```

Comment:
I can't judge for a function word's translation, so could you give us a example of what will be in %1$s, %2$s, %3$s?

Reply:
Yes please view https://zyphlar.github.io/organicmaps-locale-viewer/ in order to preview every possible word combination. This formatter is used to create all of the translations that use the words "Main Street" in that example. You can see more details in the "comment" in sounds.txt